cocos/particle-2d/particle-simulator-2d.ts

```diff
@@ -111,6 +111,7 @@ export class Simulator {
     private emitCounter = 0;
     private _worldRotation = 0;
     private declare sys;
+    public declare renderData;
```

Comment:
public shouldn't be placed after private

Reply:
> public shouldn't be placed after private

Fixed